test/__tests__/data/intercom_input.json

```diff
@@ -872,7 +872,7 @@
     },
     "destination": {
       "Config": {
-        "apiKey": "dG9rOjFkNTY4NDk1X2E3OGFfNDBiM19hYjJkXzAyMDExY2Y3YzJjZjoxOjA=",
+        "apiKey": "intercomApiKey",
         "appId": "9e9cdea1-78fa-4829-a9b2-5d7f7e96d1a0",
```

Comment:
What about this?

Reply:
Didn't change as it was not reported in github secret scanning 😅